### PR TITLE
Initial project commit.

### PR DESCRIPTION
**Pull request by LATaylor-guardian**
This commit adds the first stages of both the xml and pdf crossword uploaders.

### CODE REVIEW

File: crossword-pdf-uploader/project/build.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+sbt.version=0.13.8
```
Comment (mchv) [inline crossword-pdf-uploader/project/build.properties]: latest version is `0.13.9`

File: crossword-pdf-uploader/src/main/scala/com/gu/crossword/Config.scala [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import scala.util.Try
+
+class Config(val context: Context) {
+
+  private val isProd = Try(context.getFunctionName.toLowerCase.contains("-prod")).getOrElse(false)
+  private val stage = if (isProd) "PROD" else "CODE"
+  private val config = loadConfig()
+
+  val crosswordMicroAppUrl = Option(config.getProperty("crosswordmicroapp.url")) getOrElse sys.error("'crosswordmicroapp.url' property missing.")
+  val crosswordPdfPublicBucketName = if (isProd) "crosswords-pdf-public-PROD" else "crosswords-pdf-public"
```
Comment (mchv) [inline crossword-pdf-uploader/src/main/scala/com/gu/crossword/Config.scala]: just happening the stage will have been a bit easier I think:
 ```s""crosswords-pdf-public-${stage}""````

File: crossword-xml-uploader/project/build.properties [hunk 3]
```diff
@@ -0,0 +1 @@
+sbt.version=0.13.8
```
Comment (mchv) [inline crossword-xml-uploader/project/build.properties]: idem

**Comment by mchv** [comment]
:+1: code looks good but repo should be named `crossword-uploader`. We may use lambda in content-api components but there will be in the `content-api` repo. Hope this make sense.

**Comment by mchv** [comment]
:shipit: 

quick question about `println` do you know if they end in `cloudwatch logs`

**Comment by LATaylor-guardian** [comment]
I believe that it will log, see: http://docs.aws.amazon.com/lambda/latest/dg/java-logging.html

If not, there is a logger exposed by the lambda context that can be used.

**Pull request merged**